sdcm/cluster_aws.py

```diff
@@ -696,6 +696,9 @@ def restart(self):
 
         need_to_setup = any(ss in self._instance.instance_type for ss in ("i2", "i3", ))
 
+        if self.parent_cluster.is_ebs_volumes_attached():
+            need_to_setup = False
```

Comment:
Does it work well for "attached" on AWS?

Reply:
Yes, attached volumes are not removed during restart.